src/AcronymLike.js

```diff
@@ -32,7 +37,7 @@ const AcronymLike = (props) => {
       >
         <IconButton
           tooltip="Like"
-          onClick={() => clickHandler(isAuthenticated, liked, like, itemId, definitionId)}
+          onClick={() => clickHandler(like, definitionId, itemId, liked, isAuthenticated)}
```

Comment:
Just curious. Why did you change the order of these arguments?

Reply:
Just my ocd so they match the order where they're declared.